docs/source/technical_tutorials/working_with_r/access_cmr_r.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>read the metadata, not open the file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>